### PR TITLE
Update README.md with slightly updated GitHub Action example

### PR DESCRIPTION
**Pull request by lmagyar**
A slightly updated GitHub Action example could really help first time add-on developers (like me) to unleash the full power of this awesome project!

### CODE REVIEW

**Comment by jak119** [comment]
I came across this today while trying to build my own addon with this action and these changes would indeed make the readme more helpful.

File: README.md [hunk 1]
```diff
@@ -28,8 +28,9 @@ jobs:
         args: |
           --test \
           --all \
-          --target /data \
-          --docker-hub userspace-name
+          --target addon-folder \
+          --docker-hub user-name \
+          --image {arch}-your-addon-name
```
Comment (frenck) [inline README.md:33]: I'm confused? The image is read from the add-on configuration, right?

<https://github.com/home-assistant/builder/blob/master/builder.sh#L477>

Comment (lmagyar) [reply README.md]: Certainly you are right.
I will change this and additionally `user-name` to `${{ secrets.DOCKERHUB_USERNAME }}`

File: README.md [hunk 2]
```diff
@@ -58,8 +58,8 @@ jobs:
       with:
         args: |
           --all \
-          --target /data \
-          --docker-hub userspace-name
+          --target addon-folder \
+          --docker-hub ${{ secrets.DOCKERHUB_USERNAME }
```
Comment (frenck) [inline README.md:62]: Your DockHub username doesn't have to match the space one is publishing in?
For example; my Docker account can publish to multiple spaces.

Comment (lmagyar) [reply README.md]: Right again. Now I tried some middle-ground with `user-name-or-space-name`.
Examples are mainly for newbees like me, hard-core devs like you please fine-tune your perfectionism toward our level.

**Review by frenck — APPROVED** [review]
Thanks, @lmagyar 👍

**Pull request merged**